charts/klyshko/templates/engine-params-extra.yaml

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2023 - for information on the respective copyright owner
+# see the NOTICE file and/or the repository https://github.com/carbynestack/carbynestack.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: io.carbynestack.engine.params.extra
+data:
+  {{- $pid := (.Values.playerId | toString) -}}
+  {{- range $k, $v := .Values.engineParams.macKeyShares }}
+  {{- if ne $k $pid }}
+  {{ $k }}_mac_key_share_p: |
```

Comment:
Is there a reason the own mac key share in `io.carbynestack.engine.params.secret` are considered as secret but other parties mac keys are stored in a _ConfigMap_?

Reply:
That's due to how the Klyshko Integration Interface works. There is no other way to provide additional parameters. The setting is anyway insecure with fake tuple generation, so this doesn't seem to be a problem to me. In the future, one might consider the KII to allow for extra parameters to be a config map or a secret. But that needs additional logic in Klyshko that is out-of-scope for this PR as of my opinion.